modules/persistent-storage-local-create-cr.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-volume-cr_{context}"]
+= Provision the local volumes
+
+Local volumes can not be created by dynamic provisioning. Instead,
+PersistentVolumes must be created by the local storage provisioner. This
+provisioner will look for any devices, both Filesystem and Block volumes,
+at the paths specified in defined resource.
+
+.Prerequisites
+
+* The Local Volume Operator is installed.
+* Local disks are attached to the {product-title} nodes.
+
+.Procedure
+
+. Create the local volume resource. This must define the nodes
+and paths to the local volumes.
++
+.Example: Filesystem
+----
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "local-disks"
+  namespace: "local-storage" <1>
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+    - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values: <2>
+          - ip-10-0-140-183
+          - ip-10-0-158-139
+          - ip-10-0-164-33
+  storageClassDevices:
+    - storageClassName: "local-sc"
+      volumeMode: Filesystem <3>
+      fsType: xfs <4>
+      devicePaths: <5>
+        - /path/to/device
+----
+<1> The namespace where the Local Storage Operator is installed.
+<2> A list of nodes where the local storage volumes are attached. This example
```

Comment:
We should mention that selector is optional? If no selector is specified the operator will try to find matching disks on all nodes in the cluster.